--- CPAN::Meta::History::Meta_1_3.3pm-unpatched	2016-03-06 06:01:34.332823646 -0500
+++ CPAN::Meta::History::Meta_1_3.3pm	2016-03-06 06:02:01.244761750 -0500
@@ -140,18 +140,6 @@
 .nh
 .SH "NAME"
 CPAN::Meta::History::Meta_1_3 \- Version 1.3 metadata specification for META.yml
-.SH "PREFACE"
-.IX Header "PREFACE"
-This is a historical copy of the version 1.3 specification for \fI\s-1META\s0.yml\fR
-files, copyright by Ken Williams and licensed under the same terms as Perl
-itself.
-.PP
-Modifications from the original:
-.IP "\(bu" 4
-Various spelling corrections
-.IP "\(bu" 4
-Include list of valid licenses from Module::Build 0.2805 rather than
-linking to the module.
 .SH "SYNOPSIS"
 .IX Header "SYNOPSIS"
 .Vb 10
@@ -192,6 +180,17 @@
 \& generated_by: Module::Build version 0.20
 .Ve
 .SH "DESCRIPTION"
+.IX Header "PREFACE"
+This is a historical copy of the version 1.3 specification for \fI\s-1META\s0.yml\fR
+files, copyright by Ken Williams and licensed under the same terms as Perl
+itself.
+.PP
+Modifications from the original:
+.IP "\(bu" 4
+Various spelling corrections
+.IP "\(bu" 4
+Include list of valid licenses from Module::Build 0.2805 rather than
+linking to the module.
 .IX Header "DESCRIPTION"
 This document describes version 1.3 of the \fI\s-1META\s0.yml\fR specification.
 .PP
