--- libcaca-canvas.3caca-unpatched	2018-06-02 21:54:28.541097243 -0400
+++ libcaca-canvas.3caca	2018-06-02 21:54:48.528961403 -0400
@@ -2,15 +2,16 @@
 .ad l
 .nh
 .SH NAME
-libcaca-canvasThe libcaca canvas format (version 1) 
- \- All types are big endian\&.
+libcaca-canvas \- The libcaca canvas format (version 1) 
+.SH DESCRIPTION
+All types are big endian\&.
 .PP
 .PP
-.nf
+.EX
 struct
 {
 magic:
-   uint8_t caca_header[2];    // "\xCA\xCA"
+   uint8_t caca_header[2];    // "\exCA\exCA"
    uint8_t caca_file_type[2]; // "CV"
 
 canvas_header:
@@ -57,6 +58,6 @@
 data_extension_N:
    \&.\&.\&.                        // reserved for future use
 };
-.fi
+.EE
 .PP
  
