--- SSL_get_current_cipher.3ssl-unpatched	2018-06-05 08:37:00.800735600 -0400
+++ SSL_get_current_cipher.3ssl	2018-06-05 08:37:27.804552079 -0400
@@ -146,7 +146,7 @@
 \&
 \& const char *SSL_get_cipher_name(const SSL *s);
 \& const char *SSL_get_cipher(const SSL *s);
-\& int SSL_get_cipher_bits(const SSL *s, int *np) \e
+\& int SSL_get_cipher_bits(const SSL *s, int *np);
 \& const char *SSL_get_cipher_version(const SSL *s);
 .Ve
 .SH "DESCRIPTION"
