--- X509_SIG_getm.3ssl-unpatched	2018-06-05 01:17:22.536003744 -0400
+++ X509_SIG_getm.3ssl	2018-06-05 01:17:55.015783011 -0400
@@ -144,7 +144,7 @@
 \& void X509_SIG_get0(const X509_SIG *sig, const X509_ALGOR **palg,
 \&                    const ASN1_OCTET_STRING **pdigest);
 \& void X509_SIG_getm(X509_SIG *sig, X509_ALGOR **palg,
-\&                    ASN1_OCTET_STRING **pdigest,
+\&                    ASN1_OCTET_STRING **pdigest);
 .Ve
 .SH "DESCRIPTION"
 .IX Header "DESCRIPTION"
